Automattic/pocket-casts-android
# Deselect Chapters - What's new updates

Part of: #1807

## Description
This updates What's new version and now display What's new for Deselect Chapters.

> [!IMPORTANT]
> - Do not merge until feature needs to be enabled
> - Update the following values before merging

### TODO
1. Update `DESELECT_CHAPTERS` in `Feature.kt `
    `defaultValue=true`
    `hasFirebaseRemoteFlag=true`
     `tier = FeatureTier.Plus(ReleaseVersion(7,60))` <- If the feature is Patron-exclusive for 7.60 release. The flow has been tested as part of https://github.com/Automattic/pocket-casts-android/pull/1896
2. We may also need to change  `deselect_chapters` remote config key in `Firebase` (Internal ref: p1709746251054939/1709746242.376339-slack-C06EHRAPW12) and update it in `Feature.kt`
3. Add Changelog entry for the appropriate release when the feature needs to be enabled

## Testing Instructions
1. Install the app from a previous commit like 85f03687
2. Install the app from this PR
4. Launch the app
5. ✅ Notice that you see What's new for Deselect Chapters

## Screenshots or Screencast 
<img width=320 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/c095f6cb-9a8f-4a90-8b48-25b44e51ecbb"/>



## Checklist
 
#### I have tested any UI changes...
